spec/models/tweet_spec.rb

```diff
@@ -1,43 +1,28 @@
-require "rails_helper"
+require 'rails_helper'
 
 RSpec.describe Tweet, type: :model do
-  # pending "add some examples to (or delete) #{__FILE__}"
-  let(:user) {
-    User.create(username: "username1",
-                email: "username1@email.com",
-                name: "Firstusername",
-                lastname: "Firstuserlastname",
-                password: "username1pass",
-                password_confirmation: "username1pass")
-  }
 
-  subject {
-    described_class.new(twett_content: "Valid tweet text",
-                        user_id: user.id)
-  }
+  subject { create(:tweet) }
 
-  describe "Validations" do
-    it "is valid with valid attributes" do
-      expect(subject).to be_valid
-    end
+  describe 'Validations' do
```

Comment:
No estoy seguro si en este PR o en otro pero no estas usando shoulda matchers para validar los modelos.
```suggestion
  it { is_expected.to validate_presence_of(:user_id) }
  it { is_expected.to validate_presence_of(:twett_content) }
```

Reply:
Si, en efecto shoulda matchers se usa en el PR 36.